src/views/MenuBar.tsx

```diff
@@ -59,12 +79,22 @@ const MenuBar = ({
       </div>
     </div>
   ) : (
-    <div className="container mx-auto px-4 py-3 w-full bg-white">
+    <div className="container mx-auto px-4 my-3 w-full bg-white">
       <Header
         title="Voice Notes"
         btnContent={<HiOutlineCog />}
         handleClick={() => history.push('/settings')}
       />
+      <div className="flex flex-row mt-4 gap-x-3 items-center">
+        <Search keyword={input} setKeyword={setInput} />
+        <IconButton
+          onClick={() => setSortAscending(!sortAscending)}
+          isLoading={false}
```

Comment:
Might this be redundant? If so, perhaps `isLoading` can be assigned a default argument in `IconButton`; e.g.
```tsx
const IconButton = ({ isLoading=false, ... }) => {
  ...
}
```